config/locales/en/course/assessment/assessments.yml

```diff
@@ -56,6 +56,7 @@ en:
         form:
           autograded: 'Automatically assign grade and experience points after finalization'
           autograded_hint: 'Note that all the questions in the assessment must be auto-gradable'
+          skippable: 'Allow students to skip steps'
```

Comment:
How about just `Allow students to skip questions` ?  This is a label, we should keep it short.